### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by srishtigautam21**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (zoef-yt) [inline .vscode/settings.json]: you can remove this file it is not required It was created by the "live server" extension.

**Review by zoef-yt — CHANGES_REQUESTED** [review]
everything else look good.

**Comment by archit7448** [comment]
looks good

**Review by archit7448 — APPROVED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (zoef-yt) [reply .vscode/settings.json]: yeah you can delete that file.

Comment (srishtigautam21) [reply .vscode/settings.json]: >
made the changes

**Pull request merged**